scripts/skills/effects/rf_cheap_trick_effect.nut

```diff
@@ -0,0 +1,122 @@
+this.rf_cheap_trick_effect <- ::inherit("scripts/skills/skill", {
+	m = {
+		// Config
+		MeleeSkillModifier = 0,		// Must be set during initialization
+		RangedSkillModifier = 0,
+		DamageMult = 0.0,
+	},
+	function create()
+	{
+		this.m.ID = "effects.rf_cheap_trick";
+		this.m.Name = "Cheap Trick";
+		this.m.Description = "Unleash a quick and deceptive move to catch opponents off-guard, allowing for a higher chance of landing a hit but at the cost of power.";
+		this.m.Icon = "ui/skills/rf_cheap_trick.png";	// TODO
+		this.m.Type = ::Const.SkillType.StatusEffect;
+		this.m.IsRemovedAfterBattle = true;
+	}
+
+	function getTooltip()
+	{
+		local ret = this.skill.getTooltip();
+
+		if (this.m.MeleeSkillModifier != 0)
+		{
+			ret.push({
+				id = 10,
+				type = "text",
+				icon = "ui/icons/melee_skill.png",
+				text = ::Reforged.Mod.Tooltips.parseString(::MSU.Text.colorizeValue(this.m.MeleeSkillModifier) + " [Melee Skill|Concept.MeleeSkill]"),
+			});
+		}
+
+		if (this.m.RangedSkillModifier != 0)
+		{
+			ret.push({
+				id = 11,
+				type = "text",
+				icon = "ui/icons/ranged_skill.png",
+				text = ::Reforged.Mod.Tooltips.parseString(::MSU.Text.colorizeValue(this.m.RangedSkillModifier) + " [Ranged Skill|Concept.RangedSkill]"),
+			});
+		}
+
+		if (this.m.DamageMult != 1.0)
+		{
+			ret.push({
+				id = 12,
+				type = "text",
+				icon = "ui/icons/damage_dealt.png",
```

Comment:
Vanilla almost always uses `damage_regular.png` for damage, so I'd suggest to use that. The `damage_dealt` is also the same icon we are using for `reach`.